sdk/eventhub/azure-eventhub/tests/pyamqp_tests/synctests/test_websocket.py

```diff
@@ -0,0 +1,27 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import pytest
+
+from azure.eventhub._pyamqp import authentication, ReceiveClient
+from azure.eventhub._pyamqp.constants import TransportType
+
+def test_event_hubs_client_web_socket(live_eventhub):
```

Comment:
Async test?
And maybe some kind of test for the proxy settings?

Reply:
https://github.com/Azure/azure-uamqp-python/pull/323/files#diff-7ef7f7a671052d98771c46e57dc5b5fa98775fbe1e2e394d71df21cc25c792df

Reply:
for proxy, CI doesn't support it - running this sample locally using https://github.com/abhinavsingh/proxy.py
https://github.com/Azure/azure-sdk-for-python/blob/feature/eventhub/pyproto/sdk/eventhub/azure-eventhub/samples/sync_samples/proxy.py

Reply:
We can also use mocked offline tests for this kind of thing to at least ensure the correct code path is followed with the correct parameters.